css-three-column-layout/styles.css

```diff
@@ -0,0 +1,76 @@
+body {
+  background-color: #1996da;
+  font-family: sans-serif;
+}
+
+.container {
+  margin: auto;
+  width: 75%;
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+  flex-direction: column;
```

Comment:
Layout classes should stay unpolluted. The only styling on the `container `class should be `max-width` and `margin: auto`. ANything else should be a separate class you can add in addition to this class. That being said, I'm not sure why this would need to be a flexbox, especially a column flexbox. It seems like you are just trying to get the elements to stack vertically, which `div` elements do by default, since they are `display: block`

Reply:
I was trying to display the content to be the center of the page vertically with `justify-content`, so I thought that was the way to about so. But I have learned that it is not and will add `margin-top` to the top `row`